pkg/kubectl/cmd/drain.go

```diff
@@ -149,14 +154,14 @@ func (o *DrainOptions) SetupDrain(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	o.mapper, o.typer = o.factory.Object()
+	o.mapper, o.typer = o.factory.Object(cmdutil.GetIncludeThirdPartyAPIs(cmd))
```

Comment:
Why would this ever need to be `true`?  I don't see us trying to drain third-party resources, at least not yet.  We can't interpret their node affinity.